handlers/permissionHandlers.js

```diff
@@ -6,6 +6,7 @@ const RoleEnum = {
   Moderator: 2,
   Admin: 3,
   'Super Admin': 4,
+  'Better Then Admin': 5,
```

Comment:
I know this works for testing, but I'd prefer not to have it in the codebase just for that.  Let's take it out and have @Vic-ST make the highest role in the server be the "Super Admin" role instead.